chore: update pr and issue templates

luqven opened the pull request:
Update the PR and issue GitHub templates.

luqven committed:
chore: update pr and issue templates

luqven committed:
chore: remove broken issue link

luqven committed:
chore: create add or issue and pr gh actions

luqven committed:
chore: update pat and project id

luqven committed:
chore: set action version

sherwinski commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -1,28 +1,46 @@
 **Information:**
- - imgix-swift version: [e.g. 2.0.0]
+
+- @imgix/vue version: [e.g. v1.0]
+- browser version: [include link from [https://www.whatsmybrowser.org/](https://www.whatsmybrowser.org/) or details about the OS used and browser version]
```
It looks like we accidentally replaced swift with vue here

sherwinski reviewed (changes requested):
LGTM minus two small nits

luqven commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -1,28 +1,46 @@
+**Before you submit:**
+
+- [ ] Please read the [contributing guidelines](CONTRIBUTING.md)
+- [ ] Please search through the existing issues (both open AND closed) to see if your issue has been discussed before. Github issue search can be used for this: https://github.com/imgix/vue/issues?utf8=%E2%9C%93&q=is%3Aissue
```
```suggestion
- [ ] Please search through the existing issues (both open AND closed) to see if your issue has been discussed before. GitHub issue search can be used for this: https://github.com/imgix/imgix-swift/issues?utf8=%E2%9C%93&q=is%3Aissue
```

luqven commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -1,28 +1,46 @@
-Steps to reproduce the behavior:
+A bug is a _demonstrable problem_ that is caused by the code in the repository. Thus, the contributors need a way to reproduce your issue - if we can't reproduce your issue, we can't help you! Also, please be as detailed as possible.
+
+[a link to a codesandox or repl.it; here is a link to a codesandbox with @imgix/vue installed which can be forked: https://codesandbox.io/s/vue-imgix-base-codesandbox-bhz8n]
```
```suggestion
[add a link to a codesandox or repl.it]
```

luqven commented on .github/ISSUE_TEMPLATE/bug_report.md:
````diff
@@ -1,28 +1,46 @@
-**Expected behavior**
+```js
+// A *self-contained* demonstration of the problem follows...
+// This should be able to be dropped into a file with @imgix/vue installed and just work
````
```suggestion
// This should be able to be dropped into a file with imgix-swift installed and just work
```

luqven reviewed (commented):
🤦🏼 apols, let's replace all those `vue` with `imgix-swift`.

luqven committed:
chore: replace instances of vue with swift

This commit fixes a typo where vue was referenced instead of swift in the templates.

sherwinski commented on .github/ISSUE_TEMPLATE/bug_report.md:
````diff
@@ -1,28 +1,46 @@
+[alternatively, please provide a code example]
 
-**Expected behavior**
+```js
````
super nitpick but there's also this
```suggestion
```swift
```

luqven committed:
chore: change js reference to swift

Co-authored-by: Sherwin Heydarbeygi <sherwin@imgix.com>

luqven commented:
Apols, i didn't realize the other pr was still based on this one 😬
